django_graph_api/graphql/validation.py

```diff
@@ -0,0 +1,64 @@
+# coding: utf-8
+from graphql.ast import Variable
+
+from django_graph_api import RelatedField
+from django_graph_api.graphql.utils import GraphQLError
+
+
+def validate_args(node):
+    if node is None:
+        return
+    for field_name, field in node.fields.items():
+        for arg_name, arg in field.arguments.items():
+            validate_arg(arg_name, arg, field_name, field)
+        if isinstance(field, RelatedField):
+            obj_instance = field.get_object_instance()
+            validate_args(obj_instance)
+
+
+def get_arg_value(arg_name, field):
+    query_arguments = {arg.name: arg.value for arg in field.selection.arguments}
```

Comment:
Could refactor this to inline in the one place it's used here (while moving `resolve_variables` out) or for `types.py` to use this when getting arguments to pass to resolvers.